### PR TITLE
[excel] (File storage) Conditional Access policies and Office Scripts

### PR DESCRIPTION
**Pull request by AlexJerabek**
This PR documents the effects of Conditional Access on Office Script. Mainly, that they won't work from unmanaged devices if those policies are in place.

### CODE REVIEW

File: docs/testing/power-automate-troubleshooting.md [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
 ms.topic: troubleshooting-general
-ms.date: 04/20/2023
+ms.date: 09/08/2023
 ms.localizationpriority: medium
 ---
 
 # Troubleshoot Office Scripts running in Power Automate
 
-Power Automate lets you take your Office Script automation to the next level. However, because Power Automate runs scripts on your behalf in independent Excel sessions, there are a few important things to note.
+Power Automate runs scripts on your behalf in independent Excel sessions. This causes some behavioral changes that may create issues with certain scripts or scenarios. There are also limitations and behaviors from the Power Automate platform script writers should know. Be sure read the articles [Troubleshoot Office Scripts](troubleshooting.md) and [Platform limits and requirements with Office Scripts](platform-limits.md), as much of that information also applies to scripts in flows.
```
Comment (michelleranmsft) [inline docs/testing/power-automate-troubleshooting.md:11]: "Be sure to read"?

File: docs/testing/platform-limits.md [hunk 2]
```diff
@@ -76,6 +76,15 @@ Your browser needs third-party cookies enabled to show the **Automate** tab in E
 - [Firefox](https://support.mozilla.org/kb/disable-third-party-cookies)
 - [Safari](https://support.apple.com/guide/safari/manage-cookies-and-website-data-sfri11471/mac)
 
+## Conditional Access
+
+[Conditional Access](/azure/active-directory/conditional-access/overview) policies can restrict access to SharePoint and OneDrive for [unmanaged devices](/sharepoint/control-access-from-unmanaged-devices). If your device isn't managed by the tenant, you may not have access to specific scripts, or may only be able to access them through the browser.
+
+If you script is blocked by Conditional Access policies, you'll receive one of two error messages. These messages also surface in Power Automate if your flow is run from an unmanaged device.
+
+- "Due to organizational policies, you can’t access this resource from this untrusted device."
```
[the remark this answers is not in the document]
Comment (AlexJerabek) [reply docs/testing/platform-limits.md:85]: Is that in lieu of the other "older" error message, or in addition to it?

[the remark this answers is not in the document]
Comment (michelleranmsft) [reply docs/testing/platform-limits.md:85]: As far as we've been able to test, it seems like the only possible errors in older clients are "We can't find this script" and "Due to organizational policies"

[the remark this answers is not in the document]
Comment (AlexJerabek) [reply docs/testing/platform-limits.md:85]: Got it! I'll update the docs to only include those two. Thanks for testing!

File: docs/overview/script-storage.md [hunk 3]
```diff
@@ -11,6 +11,9 @@ Office Scripts are stored as **.osts** files in your Microsoft OneDrive or a Sha
 
 Excel only recognizes and runs a script if it's in your OneDrive folder, a Sharepoint folder, or shared with the workbook. This means Excel needs internet connectivity to access Office Scripts.
 
+> [!NOTE]
+> Admin settings for Conditional Access in OneDrive and SharePoint affect Office Scripts. For more information, see the [Conditional Access section of Platform limits and requirements with Office Scripts](../testing/platform-limits.md#conditional-access)
```
Comment (lindalu-MSFT) [inline docs/overview/script-storage.md:15]:
```suggestion
> Admin settings for Conditional Access in OneDrive and SharePoint affect Office Scripts. For more information, see the [Conditional Access section of Platform limits and requirements with Office Scripts](../testing/platform-limits.md#conditional-access).
```

**Review by lindalu-MSFT — APPROVED** [review]
Just one measly missing period!

**Comment by AlexJerabek** [comment]
Thanks @lindalu-MSFT!

**Pull request merged**